src/test/java/api/requests/RequestsAPIRetrievalTests.java

```diff
@@ -98,7 +100,11 @@ public void canGetARequestById()
     assertThat(representation.getString("status"), is("Open - Not yet filled"));
     assertThat(representation.containsKey("loan"), is(true));
     assertThat(representation.containsKey("proxy"), is(true));
-   
+    assertThat(representation.containsKey("pickupServicePoint"), is(true));
+    assertThat(representation.getJsonObject("pickupServicePoint").getString("name"),
+        is(cd1.getJson().getString("name")));
+    assertThat(representation.getJsonObject("pickupServicePoint").getString("code"),
+        is(cd1.getJson().getString("code")));
```

Comment:
This test only checks for the name and code properties. The representation also includes more properties.

Depending upon the question above, about those properties, please could the tests also include those other properties.

Reply:
Will add.